File: csharp/MapleStory.OpenAPI/Src/Dto/CharacterHyperStatPresetDTO.cs
using Newtonsoft.Json;

namespace MapleStory.OpenAPI.Dto
{
    /// <summary>
    /// 하이퍼 스탯 프리셋 정보
    /// </summary>
    public class CharacterHyperStatPresetDTO
    {
        /// <summary>
        /// 스탯 종류
        /// </summary>
        [JsonProperty("stat_type")]
        public string StatType { get; set; }

        /// <summary>
        /// 스탯 투자 포인트
        /// </summary>
        [JsonProperty("stat_point")]
        public long StatPoint { get; set; }

        /// <summary>
        /// 스탯 레벨
        /// </summary>
        [JsonProperty("stat_level")]
        public long StatLevel { get; set; }

        /// <summary>
        /// 스탯 상승량
        /// </summary>
        [JsonProperty("stat_increase")]
        public string StatIncrease { get; set; }
    }
}

File: java/src/main/java/dev/spiralmoon/maplestory/api/tms/dto/character/CharacterHyperStatPresetDTO.java
package dev.spiralmoon.maplestory.api.tms.dto.character;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

/**
 * 預設的極限屬性資訊
 */
@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
public class CharacterHyperStatPresetDTO implements dev.spiralmoon.maplestory.api.common.dto.character.CharacterHyperStatPresetDTO {

    /**
     * 能力值類型
     */
    @JsonProperty("stat_type")
    private String statType;

    /**
     * 投入於能力值的點數
     */
    @JsonProperty("stat_point")
    private Long statPoint;

    /**
     * 能力值等級
     */
    @JsonProperty("stat_level")
    private long statLevel;

    /**
     * 能力值提升量
     */
    @JsonProperty("stat_increase")
    private String statIncrease;
}
